docs/index.html

```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
 
 <meta charset="utf-8">
-<meta name="generator" content="quarto-1.0.36">
+<meta name="generator" content="quarto-1.0.18">
```

Comment:
Max, are you on an older version of quarto?

Reply:
Yes. I'll update today and commit.